Dockerfile

```diff
@@ -2,12 +2,14 @@ FROM debian:jessie
 ENV \
     EXPOSED_ROOT_DIR="/seafile" \
     SEAFILE_ROOT_DIR="/opt/seafile" \
-    SEAFILE_VERSION=6.1.1 \
+    SEAFILE_VERSION=6.2.3 \
     SEAFILE_URL_PATTERN=https://download.seadrive.org/seafile-server_VERSION_x86-64.tar.gz
 
 RUN \
     DEBIAN_FRONTEND=noninteractive \
     apt-get -y update \
+    && apt-get -y install -y ca-certificates \
```

Comment:
I don't see any beneficits to install. It adds a layer to the image and it is used only for dowloading dependencies to create this image.